### PR TITLE
feat(core): file location as parameters

### LINKED ISSUE #38: Variabilize Configuration File
Hi all, 

I need, in order to run Kafkdrop in my context, to override those properties-file location.

https://github.com/obsidiandynamics/kafdrop/blob/391b0102893ddc2829fed2558f28c4093b202082/src/main/java/kafdrop/config/KafkaConfiguration.java#L19

I can provide a PR, if you want to.

Is it OK for you ?

### PR DESCRIPTION
**Pull request by ADefossez-Zenika**
Fix #38

Allow kafka configuration, truststore and keystore files to be set via env.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -177,6 +177,18 @@ docker run -d --rm -p 9000:9000 \
     -e KAFKA_KEYSTORE=$(cat kafka.keystore.jks | base64) \       # optional
     obsidiandynamics/kafdrop
 ```
+#### Environment Variables
+
+|Name |Comment|
+|----------------------|-------------------------------|
+|KAFKA_PROPERTIES_FILE |Localtion where kafka.properties file will be written (if KAFKA_PROPERTIES is set), and read by the application|
```
Comment (ekoutanov) [inline README.md]: Typo in 'Localtion'

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -30,4 +30,7 @@ kafka:
   brokerConnect: localhost:9092
   isSecured: false
   saslMechanism: "PLAIN"
-  securityProtocol: "SASL_PLAINTEXT"
\ No newline at end of file
+  securityProtocol: "SASL_PLAINTEXT"
+  truststoreLocation : "${TRUSTSTORE_LOCATION:/kafka.truststore.jks}"
```
Comment (ekoutanov) [inline src/main/resources/application.yml]: You have `_LOCATION` at the end of the environment variable, whereas in the `README.md` and in `kafdrop.sh` there is no `_LOCATION` suffix at the end of this environment variable. Is this right?

**Review by ekoutanov — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ADefossez-Zenika) [reply README.md]: oops

File: src/main/resources/application.yml (hunk 2 above)
Comment (ADefossez-Zenika) [reply src/main/resources/application.yml]: Oh yes, my bad. I check that ;)

**Review by ADefossez-Zenika — COMMENTED** [review]
I make requested changes, TY

**Review by ekoutanov — APPROVED** [review]

**Pull request merged**